docs/wcs/index.rst

```diff
@@ -11,248 +11,114 @@ World Coordinate System (`astropy.wcs`)
 
-       - Performing `SIP`_ transformations only:
+    3. Performing `SIP`_ transformations only:
 
-         - `~astropy.wcs.wcs.WCS.sip_pix2foc`: Convert from pixel to
+        - `~astropy.wcs.wcs.WCS.sip_pix2foc`: Convert from pixel to
            focal plane coordinates using the `SIP`_ polynomial
            coefficients.
 
-         - `~astropy.wcs.wcs.WCS.sip_foc2pix`: Convert from focal
+        - `~astropy.wcs.wcs.WCS.sip_foc2pix`: Convert from focal
            plane to pixel coordinates using the `SIP`_ polynomial
            coefficients.
 
-       - Performing `distortion paper`_ transformations only:
+    4. Performing `distortion paper`_ transformations only:
 
-         - `~astropy.wcs.wcs.WCS.p4_pix2foc`: Convert from pixel to
+        - `~astropy.wcs.wcs.WCS.p4_pix2foc`: Convert from pixel to
            focal plane coordinates using the table lookup distortion
            method described in the FITS WCS `distortion paper`_.
 
-         - `~astropy.wcs.wcs.WCS.det2im`: Convert from detector
+        - `~astropy.wcs.wcs.WCS.det2im`: Convert from detector
            coordinates to image coordinates.  Commonly used for narrow
            column correction.
 
-For example, to convert pixel coordinates from a two dimensional image to world coordinates::
+For example, to convert pixel coordinates from a two dimensional image
+to world coordinates::
 
     >>> from astropy.wcs import WCS
-    >>> w = WCS('image.fits')  # doctest: +IGNORE_WARNINGS
-    >>> lon, lat = w.all_pix2world(30, 40, 0)
+    >>> wcs = WCS('image.fits')  # doctest: +IGNORE_WARNINGS
+    >>> lon, lat = wcs.all_pix2world(30, 40, 0)
     >>> print(lon, lat)
     31.0 41.0
 
+The applications of the other transformations in the above list are similar.
+
+Load from a file
+----------------
 
 Using `astropy.wcs`
 ===================
 
-Loading WCS information from a FITS file
-----------------------------------------
+High level interface
+--------------------
 
-This example loads a FITS file (supplied on the commandline) and uses
-the WCS cards in its primary header to transform.
-
-.. literalinclude:: examples/from_file.py
-   :language: python
-
-Building a WCS structure programmatically
------------------------------------------
-
-This example, rather than starting from a FITS header, sets WCS values
-programmatically, uses those settings to transform some points, and then
-saves those settings to a new FITS header.
-
-.. literalinclude:: examples/programmatic.py
-   :language: python
-
-.. note::
-    The members of the WCS object correspond roughly to the key/value
-    pairs in the FITS header.  However, they are adjusted and
-    normalized in a number of ways that make performing the WCS
-    transformation easier.  Therefore, they can not be relied upon to
-    get the original values in the header.  To build up a FITS header
-    directly and specifically, use `astropy.io.fits.Header` directly.
-
-.. _wcslint:
```

Comment:
Removing this section ref failed the `html-docs` build. FYI.

Reply:
Thanks @pllim for the heads-up.

Reply:
Section restored at least temporarily...